docker-compose.test.yml

```
@@ -39,12 +20,13 @@ services:
       network: host
     networks:
       - network
+    volumes:
+      - /mnt/.ivy2:/root/.ivy2
```

```suggestion
    volumes:
      - ./jars_cache:/root/.ivy2
```

И добавь `jars_cache` в `.gitignore` + `.dockerignore`